pkg/cmd/serve.go

```diff
@@ -169,6 +169,7 @@ func RegisterServeFlags(cmd *cobra.Command, config *server.Config) error {
 			Msg("The old implementation of LookupResources is no longer available, and a `false` value is no longer valid. Please remove this flag.")
 	}
 
+	experimentalFlags.BoolVar(&config.EnableExperimentalRevisionHeartbeat, "enable-experimental-revision-heartbeat", false, "enables experimental support for postgres revision heartbeat, used to create a synthetic revision on a given interval (postgres only)")
```

Comment:
I would lean towards this being `true` by default right away, since I think the behavior without it could be characterized as a bug if you have _any_ other workload talking to the same PG instance. As an optimization you could disable it if you're talking to a DB that is dedicated to a single spicedb, and know you don't need the watch heartbeats.

i.e. I'm not sure it's worth soaking this as default-false for a release, but I don't feel strongly if you disagree.

Reply:
I don't disagree with any of that - I was just being conservative. I think the part that I wouldn't be so ok with is the fact we are turning an experimental flag on by default, which seems to contradict the definition of experimental flag 😅 . Should I make it non-experimental?

Reply:
yeah I think I'd lean towards not experimental, and then it's something you can disable if you really want a tiny bit of extra perf from pg and know it's safe.